.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: Continuous integration
+
+on: [pull_request]
+
+jobs:
+  cypress-run:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          cache: "yarn"
+
+      - name: Install
+        run: yarn 
```

Comment:
I'm not too familiar with yarm, but I think that a similar behavior to `npm ci` would be something like `yarn install --immutable` (This was the right place)

Reply:
Yes,  yarn install --immutable is when using yarn 2, In my case I  use version 1 and it is  yarn install --frozen-lockfile